db/seeder.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable no-console */
+const db = require('./database');
+
+// The Data!
+// DEV NOTES: Consider using your own urls here. An option is to create urls from files:
+// Use Google sheets to concat names to create url and multiline edit to add quotes
+const photoUrl = [
+  'https://leileihrr.s3.amazonaws.com/livingroom002.jpg',
+  'https://leileihrr.s3.amazonaws.com/bath001.jpg',
+  'https://leileihrr.s3.amazonaws.com/bath002.jpg',
+  'https://leileihrr.s3.amazonaws.com/bath003.jpg',
+  'https://leileihrr.s3.amazonaws.com/bath004.jpg',
+  'https://leileihrr.s3.amazonaws.com/bath005.jpg',
+  'https://leileihrr.s3.amazonaws.com/bath006.jpg',
+  'https://leileihrr.s3.amazonaws.com/bed001.jpg',
+  'https://leileihrr.s3.amazonaws.com/bed002.jpg',
+  'https://leileihrr.s3.amazonaws.com/bed003.jpg',
+  'https://leileihrr.s3.amazonaws.com/bed004.jpg',
+  'https://leileihrr.s3.amazonaws.com/bed005.jpg',
+  'https://leileihrr.s3.amazonaws.com/bed006.jpg',
+  'https://leileihrr.s3.amazonaws.com/bed007.jpg',
+  'https://leileihrr.s3.amazonaws.com/bed008.jpg',
+  'https://leileihrr.s3.amazonaws.com/decor001.jpg',
+  'https://leileihrr.s3.amazonaws.com/decor002.jpg',
+  'https://leileihrr.s3.amazonaws.com/decor003.jpg',
+  'https://leileihrr.s3.amazonaws.com/decor004.jpg',
+  'https://leileihrr.s3.amazonaws.com/decor005.jpg',
+  'https://leileihrr.s3.amazonaws.com/decor006.jpg',
+  'https://leileihrr.s3.amazonaws.com/kitchen001.jpg',
+  'https://leileihrr.s3.amazonaws.com/kitchen002.jpg',
+  'https://leileihrr.s3.amazonaws.com/kitchen003.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom001.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom003.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom004.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom005.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom006.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom007.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom008.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom009.jpg',
+  'https://leileihrr.s3.amazonaws.com/livingroom010.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors001.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors002.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors003.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors4.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors5.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors6.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors7.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors8.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors9.jpg',
+  'https://leileihrr.s3.amazonaws.com/outdoors10.jpg',
+];
+const photoDesc = [
+  '',
+  'Spacious',
+  'Bright',
+  'Cheerful',
+  'Newly added',
+  'Comfortable',
+  'Cozy',
+  'Ensuite',
+  'Lofty',
+  'Warm',
+  'Roomy for the area',
+  'Accessible',
+  'Charming',
+  'Convenient',
+  'Functional',
+  'Comfy',
+  'Quaint',
+  'Soundproof',
+  'Renovated',
+  'Stylish',
+  '',
```

Comment:
Just curious.. is there a reason why you added this empty string two times (line 53 and 73)?

Reply:
Wanted the chance of blank to show up more often, and this seemed quick for the seeder.